cli/aws_ddk/commands/init.py

```diff
@@ -49,12 +55,32 @@ def init_project(name: str, environment: str, template: Optional[str]) -> None:
         },
     )
 
-    # Create virtual environment (.venv)
-    echo(f"Creating virtual environment in `{path}`...")
-    cmd: str = f"{python_exec} -m venv '{path}/.venv'"
-    try:
-        run(cmd)
-    except Exception:
-        secho(f"Failed to run `{cmd}`", blink=True, bold=True)
+    if not generate_only:
+        # Create git repository
+        if not is_in_git_repository(path):
+            echo("Initializing a new git repository...")
+            cmd_init: str = "git init"
```

Comment:
Since we don't reuse these, do we need the variables? Alternatively, how about we have a cmd = "git init && git add . && git commit..." and if there is any exception show the full command that failed?

Reply:
I can remove the use of variables but I cannot merge them into a single command, sh.py doesn't allow passing the shell=True argument. I've pushed the change to the branch.